packages/js/tracks/src/utils.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Internal dependencies
+ */
+import { EVENT_NAME_REGEX, PROP_NAME_REGEX } from './constants';
+import { ExtraProperties } from '.';
+
+export const isDevelopmentMode = process.env.NODE_ENV === 'development';
+
+export function validateEventNameAndProperties(
```

Comment:
Maybe `validateEvent` if we want to make this a bit more concise?  That said, `validateEventNameAndProperties` is a great name too and describes what this does well.

Reply:
Good idea!